src/tools/dart-run.md

```diff
@@ -0,0 +1,96 @@
+---
+title: dart run
+deprogramion: Use dart run to run a Dart program in your package.
```

Comment:
I'm assuming you meant `description` here and from the content it seems we want to describe running programs in more than just "your" package.

```suggestion
description: Use dart run to run a Dart program.
```